File: Entregables/Palao Rocio/lista_ficheros/listaFicherosJava/src/main/java/org/example/models/Usuario.java
package org.example.models;

import java.io.Serializable;
import java.util.UUID;

public class Usuario implements Serializable {
    UUID id;
    String nombre;
    String email;

    public Usuario(UUID id, String nombre, String email) {
        this.id = id;
        this.nombre = nombre;
        this.email = email;
    }

    public String toCsv(){
        return id+";"+nombre+";"+email;
    }

    @Override
    public String toString() {
        return "Usuario{" +
                "id=" + id +
                ", nombre='" + nombre + '\'' +
                ", email='" + email + '\'' +
                '}';
    }
}

File: Entregables/JorgeSanchezBerrocoso/EscribirLeerFicheros/src/main/kotlin/Models/Usuario.kt
package Models

import java.io.Serializable
import java.util.*

class Usuario(val id: String = UUID.randomUUID().toString(), var nombre : String?,var correo: String?) : Serializable {
  init {
      println("Se ha creado el usuario")
  }
    override fun toString(): String{
        return "Usuario(uuid: $id, nombre: $nombre, correo: $correo)"
    }
}